docs/crates/vm.md

```diff
@@ -115,35 +228,34 @@ which in turn provides the list of AIRs that are used in the proving and verific
 
```

Comment:
please update keygen above: it doesn't need `VmChipComplex` and only `create_airs()`